README.md

```diff
@@ -1 +1,185 @@
-# Leaderboard
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <h1><b>LEADERBOARD</b></h1>
+  <br />
+  <img src="./src/leaderboard.png" alt="screenshot" width="78%"  height="78%" />
+  <br />
+</div>
+
+<br />
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 About ](#-about-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)

```

Comment:
- Since, you've not deployed the app, let's remove this also as the link is currently broken.